src/engine/controls/loopingcontrol.cpp

```diff
@@ -345,7 +345,8 @@ void LoopingControl::process(const double dRate,
 
 double LoopingControl::nextTrigger(bool reverse,
         const double currentSample,
-        double *pTarget) {
+        double* pTarget,
+        bool repeat) {
```

Comment:
After merging to 2.4 I'd add a PollingControlProxy to undo all this passing around of `repeat`.
Or we use a ControlProxy right away and replace that in 2.4 (that's probably better / cleaner because I can revert all scaer and test changes).

Reply:
Done. Much simpler now.